### PR TITLE
libsrtp 2.5.0

### PR DESCRIPTION
**Pull request by piliugin-anton**
Specify library name and version:  **libsrtp/2.5.0**

Recipe update

---

- [x] I've read the [contributing guidelines](https://github.com/conan-io/conan-center-index/blob/master/CONTRIBUTING.md).
- [x] I've used a [recent](https://github.com/conan-io/conan/releases/latest) Conan client version close to the [currently deployed](https://github.com/conan-io/conan-center-index/blob/master/.c3i/config_v1.yml#L6).
- [ ] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Comment by piliugin-anton** [comment]
@toge @uilianries Could someone explain what's with CI? I see "Pending — This commit is being built" message for an hour or more

File: recipes/libsrtp/all/conanfile.py
Comment (RubenRBS) [inline recipes/libsrtp/all/conanfile.py:60]: This is missed a pinned version for the packages

**Review by RubenRBS — CHANGES_REQUESTED** [review]
Thanks! Some changes are needed before this can be merged

File: recipes/libsrtp/all/conanfile.py
Comment (piliugin-anton) [reply recipes/libsrtp/all/conanfile.py]: Added required versions

File: recipes/libsrtp/all/conanfile.py
Comment (valgur) [inline recipes/libsrtp/all/conanfile.py:61]:
```suggestion
        if self.options.with_nss:
            self.requires("nss/3.93")

        if self.options.with_mbedtls:
            self.requires("mbedtls/3.5.2")

```
Version ranges are only allowed for a small subset of dependencies on CCI currently, unfortunately.

**Review by valgur — CHANGES_REQUESTED** [review]
LGTM. Just one minor change needed for the version ranges.

**Comment by valgur** [comment]
Blocked by #19262.

**Comment by ericLemanissier** [comment]
@RubenRBS this can be closed thanks to https://github.com/conan-io/conan-center-index/commit/ae2808e8b6b149beb845d4a4e632b9ed2fe8c8c1

**Pull request closed without merging**
